tests/models/bloom/test_modeling_bloom.py

```diff
@@ -377,15 +377,34 @@ def test_model_from_pretrained(self):
     @slow
     @require_torch_gpu
     def test_simple_generation(self):
+        # This test is a bit flaky. For some GPU architectures, pytorch sets by default allow_fp16_reduced_precision_reduction = True and some operations
+        # do not give the same results under this configuration, especially torch.baddmm and torch.bmm. https://pytorch.org/docs/stable/notes/numerical_accuracy.html#fp16-on-mi200
+        # We set allow_fp16_reduced_precision_reduction = True. Please see: https://pytorch.org/docs/stable/notes/cuda.html#reduced-precision-reduction-in-fp16-gemms
```

Comment:
I am just wondering what this statement for: `We set allow_fp16_reduced_precision_reduction = True.`

Reply:
I believe we meant to say that we leave allow_fp16_reduced_precision_reduction = True, as set by default. But I agree with you that the sentence could do some rephrasing. Wdyt @younesbelkada?

Reply:
Yeah agree that the sentence is a bit confusing, we should say something like
`Therefore we should set allow_fp16_reduced_precision_reduction = True`

Reply:
If we use `should`, it looks like there are some reasons, and it would be better to include it.

However, If I understand correctly, what you mean here is more about

*As we leave the default value (`True`) for `allow_fp16_reduced_precision_reduction` , the tests failed when running in half-precision with smaller models (350m)*

(just a description of the situation, and the fix we take is to run in fp32)

Reply:
Exact, yes totally agree with your proposal. Let's just describe the situation !

Reply:
It's more of wording thing, I will leave this for you two to decide and open a PR if necessary. My only goal is to avoid confusion for the readers